neurots/generate/tree.py

```diff
@@ -120,6 +120,7 @@ def __init__(
         self.active_sections = []
         self.context = context
         self._rng = random_generator
+        self.two_major = parameters.get("two_major", False)
```

Comment:
this is to have two sub-apical tree that bifurcate from the first bif after soma, and both go up. Without this, one will be a trunk, and the other will go on the side as an oblique. This is specific to the hipp PC cells, but may be found elsewhere?

Reply:
Sometimes its not 2 major but 3 or even 4 depending on the species. 
![image](https://github.com/BlueBrain/NeuroTS/assets/29337051/30e4217a-e45b-49e7-8c57-b9a487b57984)
and neocortex also have these morphologies.